hexrd/instrument/detector.py

```diff
@@ -152,7 +149,7 @@ def beam_position(self):
         returns the coordinates of the beam in the cartesian detector
         frame {Xd, Yd, Zd}.  NaNs if no intersection.
         """
-        pass
```

Comment:
This pass is necessary, please either do not remove it, or raise a `NotImplementedError`.